languages/ruby/exercises/concept/arrays/.docs/after.md

```diff
@@ -0,0 +1,54 @@
+Data structures that can hold zero or more elements are known as _collections_. An **array** in Ruby is a collection that maintains the ordering in which its objects are added. Arrays can hold any object. Objects can be added to an array or retrieved from it using an index. Ruby array indexing is zero-based, meaning that the first element's index is always zero:
```

Comment:
> Arrays can hold any object

Should that be `Arrays can hold any type of object`?

Reply:
No. I think this is correct. What do you not like about it?